src/tsf/TfConvArea.h

```diff
@@ -31,15 +31,15 @@ class CConversionArea
 {
 public:
     [[nodiscard]]
-    HRESULT DrawComposition(const CComBSTR& CompStr,
+    HRESULT DrawComposition(const std::wstring_view& CompStr,

```

Comment:
Hm. Usually I pass constant `std::wstring_view`s by value instead of by reference to 100% commit to the fact that they can't change anything. I suppose that might use a bit more memory though. (pointer + size_t for value instead of just pointer for ref.) Was this just a mechanical substitution?